package.json

```diff
@@ -24,7 +24,9 @@
     "eslint": "^7.32.0",
     "eslint-config-airbnb-base": "^14.2.1",
     "eslint-plugin-import": "^2.26.0",
+    "file-loader": "^6.2.0",
     "hint": "^7.0.1",
+    "html-loader": "^3.1.2",
     "html-webpack-plugin": "^5.5.0",
     "style-loader": "^3.3.1",
     "stylelint": "^13.13.1",
```

Comment:
- [ ]  You should set the right command in your script/package.json to run your project successfully. and install `webpack-dev-server` so that once the project succeeds, It will auto run successfully. 
![image](https://user-images.githubusercontent.com/43877689/177334571-1209d3a2-51ee-46c8-b88c-3fe8f01acaa8.png)